### DISCUSSION TITLE
Auto Sync when changing cockpit driver stream

### DISCUSSION THREAD

**Discussion by RevengeFNF** [f1multiviewer/issue-tracker#330] [Feature requests]
When we change the stream of a cockpit driver to another one, that stream will start from the beginning and we need to sync all streams again.
Would it be possible that when we do that, it will sync automatically?

**Comment by jeppeolesen** [comment]
This should be implemented in the later versions of MultiViewer. If it's still an issue, please file a bug report under issues: https://github.com/f1multiviewer/issue-tracker/issues
